import-service/handlers/import-file-parser/copy-and-delete-s3-obj.ts

```diff
@@ -0,0 +1,24 @@
+import 'source-map-support/register';
```

Comment:
I suggest to move this file into `utils` folder. So in `handlers` folder you will have only two lambdas

Reply:
done